cdk.json

```diff
@@ -43,6 +43,19 @@
       "STICKY":"TRUE",
       "DESIRED_TASK_COUNT":"2"
     },
+    "staging": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:0.4.8-beta",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "TAGS": {
+        "CostCenter": "NO PROGRAM / 000000"
+      },
+      "STACK_NAME_PREFIX": "dca-staging",
+      "ACM_CERT_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/a3b7c804-c1fc-4e58-bbc6-541ef2d65816",
+      "VPC_CIDR": "10.255.71.0/24",
```

Comment:
I think this will need a unique CIDR range

Reply:
Is this something I can change, like 10.255.72.0/24?

Reply:
10.255.72.0/24 used to belong to dca-no-sticky. Looks like schematic goes from 10.255.73-75. Are CNAMES stored someplace centrally to avoid accidentally overwriting them?

Reply:
Yeah 10.255.72.0/24 will work, I used the directions at https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2850586648/Setup+AWS+VPC to confirm that it isn't currently overlapping with any existing VPCs